src/LibraryUtilities.ts

```diff
@@ -203,6 +248,23 @@ export function constructLibraryItem(
     return result;
 }
 
+// E.g. Consider the following scenario with each variable value being:
+// "fullyQualifiedName" = "DSCore.Math.RandomList"
+// "includePath" = DSCore.Math.Random
+// "fullyQualifiedNameParts" = [ "DSCore", "Math", "RandomList" ]
+// "includeParts" = [ "DSCore", "Math", "Random" ]
+// Comparing fullyQualifiedName and includePath with only startsWith() will return true,
+// which is undesired since Random and RandomList are two different nodes.
+// Further checking is required to match the two strings using matchedUpTo().
+function matchedUpTo(fullyQualifiedNameParts: string[], includeParts: string[]): boolean {
```

Comment:
Documentation of an API should not include things that are irrelevant (e.g. why does the reader need to know `startsWith`?). We should simply give scenarios that will make the API returns `true` or `false`, by reading the documentation its usage should become really clear.

Also please use the documentation format that is similar to the following `convertToLibraryTree` API.